packages/web-react/scripts/entryPoints.js

```diff
@@ -24,7 +24,38 @@ exports.map = function map(callback, context) {
 
 const pathPosix = require('path').posix;
 
-exports.check = function (id, parentId) {
+function lengthOfLongestEntryPoint(parts) {
+  let node = lookupTrie;
+  let longest = -1;
+  for (let i = 0; node && i < parts.length; ++i) {
+    if (node.isEntry) longest = i;
+    node = node.dirs && node.dirs[parts[i]];
+  }
+  if (node && node.isEntry) {
+    return parts.length;
+  }
+
+  return longest;
+}
+
+// eslint-disable-next-line consistent-return
+function partsAfterDist(id) {
+  const parts = id.split(pathPosix.sep);
+  const distIndex = parts.lastIndexOf('dist');
+  if (distIndex >= 0) {
+    return parts.slice(distIndex + 1);
+  }
+}
```

Comment:
```suggestion
function partsAfterDist(id) {
  const parts = id.split(pathPosix.sep);
  const distIndex = parts.lastIndexOf('dist');
  if (distIndex >= 0) {
    return parts.slice(distIndex + 1);
  }
  return null;
}
```

… would't work?

Reply:
I gave it a try and seems to be working.